PluralKit.Bot/Commands/Misc.cs

```diff
@@ -214,9 +228,29 @@ public async Task PermCheckGuild(Context ctx)
                     eb.Color(DiscordUtils.Red);
                 }
             }
+            string footer = "";
+
+            foreach (var (missingPermissionField, channels) in permissionsMissing)
+            {
+                if (missingPermissionField == (ulong)PermissionSet.UseExternalEmojis && footer.Length < 1)
+                {
+                    footer = "Use External Emojis permissions must be granted to the @everyone/default role.";
+                }
+            }
 
             if (hiddenChannels > 0)
-                eb.Footer(new($"{"channel".ToQuantity(hiddenChannels)} were ignored as you do not have view access to them."));
+            {   
+                if (footer.Length > 0)
+                {
+                    footer = $"{"channel".ToQuantity(hiddenChannels)} were ignored as you do not have view access to them.";
+                }
+                else
+                {
+                    footer += $"\n{"channel".ToQuantity(hiddenChannels)} were ignored as you do not have view access to them.";
+                }
```

Comment:
I wouldn't leak the amount of hidden channels, actually?

I know this code was here already but it was never a good idea in the first place :)

Reply:
does this message need to be here at all? I can only think of it being useful if the bot's cache is broken and it thinks the user is missing permissions when they actually aren't

Reply:
Yeah, maybe not? We probably don't lose anything from just removing it outright